decidim-proposals/app/queries/decidim/proposals/metrics/proposal_followers_metric_measure.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Metrics
+      class ProposalFollowersMetricMeasure < Decidim::MetricMeasure
+        # Searches for unique Users following the next objects
+        #  - Proposals
+        #  - CollaborativeDrafts
+
+        def valid?
+          super && @resource.is_a?(Decidim::Component)
+        end
+
+        def calculate
+          proposals = Decidim::Proposals::Proposal.where(component: @resource).joins(:component)
+                                                  .except_withdrawn
+          collaborative_drafts = Decidim::Proposals::CollaborativeDraft.where(component: @resource).joins(:component).except_withdrawn
+
+          proposals_followers = Decidim::Follow.where(followable: proposals).joins(:user)
+                                               .where("decidim_follows.created_at <= ?", end_time)
+          drafts_followers = Decidim::Follow.where(followable: collaborative_drafts).joins(:user)
+                                            .where("decidim_follows.created_at <= ?", end_time)
+
+          cumulative_users = []
+          cumulative_users |= proposals_followers.pluck(:decidim_user_id)
```

Comment:
What's this syntax? 😕

Reply:
Union of two arrays

Reply:
Ah, lol. I'd never seen it before and thought it was a typo of `||=`, 👍